pupil_src/shared_modules/video_capture/realsense2_backend.py

```diff
@@ -680,15 +724,13 @@ def depth_frame_size(self):
 
     @depth_frame_size.setter
     def depth_frame_size(self, new_size):
-        logger.debug("set depth_frame_size")
-        if self.pipeline is not None and new_size != self.depth_frame_size:
+        if new_size != self.depth_frame_size:
```

Comment:
Is `new_size` a `list` or a `tuple`? This might effect the proper functionality of this comparison.

Reply:
It is always a `tuple`, since frame sizes are used as keys in dictionaries (e.g. `self._available_modes`)